### PR TITLE
Update deps (latest) on tinylicious in the client release group

### PR DESCRIPTION
**Pull request by rohandubal**
Updated the following:

  client (release group)

Dependencies on tinylicious updated:

  tinylicious: 0.6.0

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -22672,15 +22830,15 @@ packages:
     resolution: {integrity: sha512-DNK4ruAqtyHaN8Zne7PkBTO+dD1Lr0YfTduMqlIyjvQIoztBkUxrvL+hKeLW8NXFKHOq/2upkxuoS9znQ9bW9A==}
     deprecated: This version of 'buffer' is out-of-date. You must update to v4.9.2 or newer
     dependencies:
-      base64-js: 1.5.1
+      base64-js: 1.3.0
```
Comment (alexvy86) [inline pnpm-lock.yaml:22833]: Can't quite make sense of why this went to a previous version. Dependencies are probably still satisfied but this shouldn't have happened with the change we made for tinylicious?

**Comment by alexvy86** [comment]
Seems like this is missing a `pnpm i` at the root, or maybe a merge conflict got resolved in a funky way. `package.json` for @fluidframework/test-end-to-end-tests declares the updated dependency on tinylicious 0.6.0 but the lockfile says it wants 0.5.0.

**Comment by alexvy86** [comment]
Oh, seems like this might have been superseded by #14494

**Comment by rohandubal** [comment]
Thanks for pointing out @alexvy86 

Yeah, I realized after a bit :( Will reach out to understand more the conflict in work items.

**Pull request closed without merging**